.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python Debugger: Module",
+            "type": "debugpy",
+            "console": "integratedTerminal",
+            "request": "launch",
+            "module": "anta.cli._main",
```

Comment:
There is 2 things here:

- Module is not built to be executed on its own. So to use `anta.cli` only, we need to rename `_main.py` to `__main__.py`

- To execute module, we need to get a default `if __name__ == "__main__"` stanza to load the correct entrypoint.

The thing is VScode debugger uses module approach to monitor code execution, so if we want to get access to it, we need to format our CLI module to be compliant with Python standard structure.

Reply:
it is there: https://github.com/aristanetworks/anta/blob/main/anta/cli/__init__.py#L40

Reply:
```
ANTA_DEBUG=True python anta/cli/__init__.py
The ANTA command line client could not run because the required dependencies were not installed.
Make sure you've installed everything with: pip install 'anta[cli]'
The caught exception was: attempted relative import with no known parent package
```

ok it is unhappy lets fix our code rather than adding extra entrypoints where we bypass validation

suggested fix in  https://github.com/aristanetworks/anta/pull/909

Reply:
> Module is not built to be executed on its own. So to use anta.cli only, we need to rename _main.py to __main__.py
we don't want this as it bypasses the check for cli dependencies being installed which is not great.


> To execute module, we need to get a default if __name__ == "__main__" stanza to load the correct entrypoint.
we have this in __init__.py but it makes python not happy ... so lets fix it!